modules/search/instant-search/components/search-app.jsx

```diff
@@ -35,23 +36,21 @@ class SearchApp extends Component {
 		this.props.resultFormat = 'minimal';
 		this.props.aggregations = buildFilterAggregations( this.props.options.widgets );
 		this.props.widgets = this.props.options.widgets ? this.props.options.widgets : [];
+		this.isSearchPage = this.props.initialValue !== '';
 		this.state = {
 			query: this.props.initialValue,
 			sort: this.props.initialSort,
 			results: {},
 			loading: false,
 		};
-		this.getResults = debounce( this.getResults, 200 );
+		this.getDebouncedResults = debounce( this.getResults, 200 );
```

Comment:
Note: this is so we can make the UI faster when the user filters/sorts and don't add the extra delay for those cases.